tests/testthat/_snaps/check_suggested.md

```diff
@@ -37,8 +37,16 @@
       Error:
       ! The package "iurtnkjvmomweicopbt" is required to use `foo()`.
     Code
-      check_suggested("iurtnkjvmomweicopbt", TRUE, message = "not installed!")
+      check_suggested("iurtnkjvmomweicopbt", TRUE)
     Condition
       Error:
-      ! The package "iurtnkjvmomweicopbt" is required not installed!
+      ! The package "iurtnkjvmomweicopbt" is required.
```

Comment:
The quotes used here are inconsistent with the quotes in the other error message.

Reply:
Will use `"` everywhere.